gravitee-apim-rest-api/postman/scripts/tests.sh

```diff
@@ -84,4 +84,9 @@ fi
 set -e;
 
 # Run all collections
-for f in $POSTMAN_DIR/test/*;do if [[ -f $f ]]; then newman run $f -e $POSTMAN_DIR/env/Gravitee.io-$POSTMAN_ENV-Environment.json --bail; fi; done;
\ No newline at end of file
+for f in $POSTMAN_DIR/test/*;do
+  if [ -f "$f" ]; then
+     echo "Run Newman tests on $f"
+     newman run $f -e $POSTMAN_DIR/env/Gravitee.io-$POSTMAN_ENV-Environment.json --bail --reporters cli --reporter-cli-no-assertions;
```

Comment:
I would have passed `--reporter-cli-no-assertions` option thanks to a "CLI" mode when running these tests.

It's just my opinion, but if I want to run postman locally, I like to have all the traces to understand if it fails somewhere.

Here, we lose some details just to adapt to the CI.
@phiz71 @gaetanmaisse I would like to know your thought about this

Reply:
True, I added it to check something but I'm not sure it's useful, I've removed it